.github/mergeable.yml

```diff
@@ -0,0 +1,269 @@
+version: 2
+mergeable:
+  # Let's welcome contributors.
+  - when: pull_request.opened
+    name: "Hola!"
+    validate: []
+    filter:
+      - do: author
+        must_exclude:
+          regex: 'dependabot'
+    pass:
+      - do: comment
+        payload:
+          body: >
```

Comment:
It seems this comment will be placed, including for all existing contributors (like our internal devs) while that might be OK I doubt the messaging in it's current format make sense

e.g. A maintainer from the @goalgorilla/maintainers group might not review all changes from all teams / contributors.

Reply:
I will change the message.

Reply:
I have updated it.